acceleration.md

```diff
@@ -0,0 +1,62 @@
+Hardware Acceleration
+=====================
+
+Ops can utilize your CPU's virtualization extension through the various
+supported hypervisors. Ops will only attempt to enable acceleration on systems
+that support it.
+
+To have Ops enable hardware acceleration when running an image, first check to
+see if your system's chip has the appropriate support. An easy way to do this on
+modern Linux based systems is to run the following command:
+
+```sh
+$ grep -woE 'svm|vmx' /proc/cpuinfo | uniq
+```
+
+If you have a supported Intel CPU you should see the output `vmx`, for supported
+AMD processors you should see `svm`. If your CPU does not support a
+virtualization extension you will see no output. If you are certain that your
+CPU provides virtualization support and yet you see no output, then please
+ensure that the extension is enabled in your BIOS/UEFI system.
+
+If using KVM you need to ensure that your user is a member of the `kvm` group.
+Ops will not attempt to enable acceleration if your user is not a member of this
+group. First check if your user is apart of the kvm group:
+
+```sh
+$ groups
+```
+
+If your user is a member of the group you should see `kvm` in the list. If not
+already a member you can add yourself to the group with the following command:
+
+```sh
+$ usermod -aG kvm `whoami`
+```
+
+The change will take affect upon the next login. Finally, you can check to see
+if Ops is using virtualization support by inspecting the command it uses to run
+an image. You can do this by enabling verbose output when using the `run` or
+`load` command. So assuming I have hardware acceleration enabled in my run-time
+configuration:
+
+```json
+{
+    "RunConfig": {
+        "UseKvm": true
```

Comment:
Perhaps the run-time config option "UseKvm" should be changed to something more general. "EnableAccel" perhaps. Remember, on Mac `hvf` is enabled with this flag not `kvm`

Reply:
that prob. makes sense although could just be shortened to 'accel' or something

Reply:
Oh, bummer, this was done but the doc wasn't changed. I'll update it. My fault.